/vlm_vision:
  ros__parameters:
    inference_hz: 3.0
    image_topic: /camera/color/image_raw
    color_image_topic: /camera/color/image_raw
    depth_image_topic: /camera/depth/image_rect_raw
    camera_info: /camera/color/camera_info
    box_threshold: 0.35
    text_threshold: 0.25
    text_prompt: "Go to the blue backpack."
    